MetaMask/metamask-docs
# feat(ref): added rpc-parser component to linea ref pages

## Description

- Added **ParserOpenRPC** component for **Linea** reference pages **services/reference/linea/json-rpc-methods** ( `/eth_accounts` , `/eth_blocknumber` , `/eth_chainid` , `/eth_gasprice` , `/eth_maxpriorityfeepergas` , `/eth_syncing` )

## Test

- Go to pages and check the display of the new layout (content part / interactive part)
- To test the content => compare the content on the page with the corresponding method in the source JSON-RPC (https://sot-network-methods.vercel.app/specs/linea)
- To test the interactive part (send request):
  - need logged in to the test env https://infura-app-staging.vercel.app/ and that you have the Linea-mainnet network enabled
   - connect from reference page & check that your API key is displayed correctly
  
![Screenshot 2024-10-01 at 13 29 17](https://github.com/user-attachments/assets/2ebe2c0c-5620-41ec-aed5-2a7b7383858f)
